wallet-new/src/Cardano/Wallet/Util.hs

```diff
@@ -27,7 +27,7 @@ headToLower []     = Nothing
 headToLower (x:xs) = Just $ toLower x : xs
 
 stripFieldPrefix :: String -> String
-stripFieldPrefix = dropWhile (not . isUpper)
+stripFieldPrefix = dropWhile (not . isUpper) . drop 1
```

Comment:
NOTE: I took some freedom in adjusting the way _I think it should_ work. It was dead-code until now though, not used anywhere, so this change is free of any impact :+1: